src/main/java/pl/sdacademy/carrental/configuration/DomainValues.java

```diff
@@ -0,0 +1,11 @@
+package pl.sdacademy.carrental.configuration;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
definately not a component :)

Reply:
indeed it is not (well, not anymore) :-)